src/views/RecIM/views/home/index.js

```diff
@@ -47,7 +47,7 @@ let myTeams = (
   </Card>
 );
 
-const Home = () => {
+const RecIM = () => {
```

Comment:
This got changed to RecIM, it needs to be home and exported as such at the bottom.